### ISSUE TITLE
Please provide details when training on cifar10

### ISSUE THREAD

**Issue by ALEX13679173326** [Sara-Ahmed/SiT#15]
Good job!! 
Could you provide some hyper-parameters for training SSL on cifar10?(just like LR,optimizer,weight decay etc)I'm curious about how to train vit on cifar10 instead of fine tuning.

**Comment by Sara-Ahmed** [comment]
Dear Alex, thanks for your interest in our work. We did not do any hyper-parameter tuning. The obtained accuracy were by using the provided default parameters.
